refresh of who do you want to teach (#3140)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

---
title: Who do you want to teach
heading: Who do you want to teach?
description: |-
  Explore who’d you’d like to teach and the qualifications you need to work with early years, further education, and pupils with special educational needs.
promo_content:
    - content/train-to-be-a-teacher/promos/mailing-list-promo-who-do-you-want-to-teach
calls_to_action:
    get-school-experience:
      name: simple
      arguments:
        icon: "icon-arrow"
        text: |-
          Use our service to search for and organise a primary or secondary school placement in England.        
        link_text: "Get school experience"
        link_target: "https://schoolexperience.education.gov.uk/"
---

It’s good to consider what kind of teacher you want to be based on your interests, skills and experience. You might need specific training or qualifications in some areas. This page explains more about the different options.

## Teach in a primary or secondary school

The school system in England is split into age groups:

### Primary (ages 5 to 11)

> Children usually start primary school at 4 years old and this phase is called Reception. This is the final stage of the Early Years Foundation Stage (EYFS).

Reviewer: GOV.UK uses `reception` here: https://www.gov.uk/government/collections/reception-baseline

There are a further 2 phases, known as key stages: 

> * key stage 1 covers year 1 and 2, with children up to 7 years old

Reviewer: ```suggestion
* key stage 1 covers years 1 and 2, with children up to 7 years old
```

> * key stage 1 covers years 1 and 2, with children up to 7 years old

Reviewer 2: ```suggestion
* key stage 1 (KS1) covers years 1 and 2, with children up to 7 years old
```
just so it's the same as the secondary section?

> * key stage 2 covers years 3 to 6, with children from 7 to 11 years old

Reviewer 2: ```suggestion
* key stage 2 (KS2) covers years 3 to 6, with children from 7 to 11 years old
```
just so it's the same as the secondary section?

The teaching content is broad, covering the whole range of national curriculum subjects including English, maths, science, as well as art and design, computing, geography, history, music and physical education (PE).

If you like the energy and curiosity of young children, and would be comfortable covering a wide range of subjects, you might prefer teaching in primary schools. 

If you are interested in training to be a primary teacher, you could consider options for primary teaching with a subject specialism like maths or English. This will cover all of the subjects but with a focus on your subject specialisation.

### Secondary (ages 11 to 18)
Children start secondary school when they’re 11. There are up to 3 stages in secondary school: 

> * key stage 3 (KS3) includes years 7 to 9, with children age 11 to 14  

Reviewer: should this be 'aged'?

> * key stage 4 (KS4) includes years 10 and 11, with pupils age 14 to 16

Reviewer: should this be 'aged'?

> * key stage 5 (KS5) includes years 12 and 13, from age 16 to 18 at secondary schools with sixth forms or colleges

Reviewer: can we say 'with students aged...' to be consistent with the other bullets? seems odd not to be consistent with children/pupils mentioned in the other ones

> * key stage 5 (KS5) includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms or colleges

Reviewer: ```suggestion
* key stage 5 (KS5) includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms, or colleges
```

Not sure whether this is stylistically acceptable, to have a comma here (there may be a better solution) - but this read to me as 'secondary schools with colleges' rather than separate entities

Pupils work towards national qualifications, usually GCSEs during KS4. Students take A levels or other similar qualifications during KS5. 

> Subjects at this level include English, maths and science. Additional subjects may include drama, dance, and media studies. The specific subjects offered will depend on the school.

Reviewer: I think this is really a bit odd now - why pick out drama, dance and media studies, rather than geography, history, art, etc.?

> Subjects at this level include English, maths and science. Additional subjects may include history, geography, and art. The specific subjects offered will depend on the school.

Reviewer: @philspray @EllieNodder I still don't think this paragraph reads quite right still. It implies that history, geography and art (which should be art and design) are possibly optional depending on the school - but aren't they compulsory in the national curriculum for KS3 at least, along with others? 
https://www.gov.uk/government/publications/national-curriculum-in-england-secondary-curriculum

I appreciate I probably muddied the waters with my specific comment previously about drama etc. but I still think this needs rewording somehow?

> Subjects at this level include English, maths, science, as well as art and design, computing, geography, history, music and physical education (PE). After the age of 14, subjects in arts, design and technology, the humanities, and modern foreign languages are not compulsory but schools must provide at least one course in each of the four areas.  

Reviewer 2: ```suggestion
Subjects at this level include core subjects such as English, maths and science, as well as many others. The full range of subjects available to teach at secondary level will depend on the school.
```
I know we all keep going back and forth on this, but I am just a bit worried about listing a load of subjects without listing _all_ the subjects, just in case someone reading that might go 'oh, I guess my subject isn't really taught at this level then cos it's not on that list.'

I've put this suggestion in here (which still might not be quite right!) but maybe simplifies it a bit?

Also, not sure if we really need that second part - feels like that's maybe guidance for the school rather than someone interested in teaching?

thoughts @philspray @gemmadallmandfe ?

Author: Would it be better to add in the missing subjects and remove the second part?

Reviewer 2: Do we _know_ all the missing subjects? haha
I'm not sure we do, so might be safer to just not have them?
But your call!

> heading: Who do you want to teach

Reviewer 2: ```suggestion
heading: Who do you want to teach?
```

description: |-
  Explore who’d you’d like to teach and the qualifications you need to work with early years, further education, and pupils with special educational needs.
promo_content:
    - content/train-to-be-a-teacher/promos/mailing-list-promo-who-do-you-want-to-teach
calls_to_action:
    get-school-experience:
      name: simple
      arguments:
        icon: "icon-arrow"
        text: |-

>           Use our service to search for and organise a placement in England.

Reviewer: ```suggestion
          Use our service to search for and organise a primary or secondary school placement in England.
```

        link_text: "Get school experience"
        link_target: "https://schoolexperience.education.gov.uk/"
---

It’s good to consider what kind of teacher you want to be based on your interests, skills and experience. You might need specific training or qualifications in some areas. This page explains more about the different options.

## Teach in a primary or secondary school

The school system in England is split into age groups:

### Primary (ages 5 to 11)
Children usually start primary school at 4 years old and this phase is called Reception. This is the final stage of the Early Years Foundation Stage (EYFS).

There are a further 2 phases, known as key stages: 

> * key stage 1 covers year 1 and 2, up to 7 years old

Reviewer: Should we be a bit more consistent with how we write these bullets in the primary and secondary sections?

eg include KS1 and KS2 notation?

eg make sure all the bullets are worded consistently re years and ages? for example

* key stage x (KSx) covers years x to x with children aged x to x (changed to pupils/students when appropriate)

> * key stage 2 covers year 3 to 6 with children from 7 to 11 years old

Reviewer 2: ```suggestion
* key stage 2 covers year 3 to 6, with children from 7 to 11 years old
```

The teaching content is broad, covering the whole range of national curriculum subjects including English, maths, science, as well as art and design, computing, geography, history, music and physical education (PE).

If you like the energy and curiosity of young children, and would be comfortable covering a wide range of subjects, you might prefer teaching in primary schools. 

> If you’re doing initial teacher training (ITT), you could consider options for primary teaching with a subject specialism like maths or English. This will cover all of the subjects but with a focus on your subject specialisation.

Reviewer: Does this make a bit of leap - they won't be doing ITT yet and might not know what that is?

More like 'If you are interested in training to be a primary teacher, .....'?

### Secondary (ages 11 to 18)
Children start secondary school when they’re 11. There are up to 3 stages in secondary school: 

> * key stage 3 (KS3) includes years 7 to 9 with children age 11 to 14  

Reviewer 2: ```suggestion
* key stage 3 (KS3) includes years 7 to 9, with children age 11 to 14  
```

> * key stage 4 (KS4) includes years 10 and 11 with pupils age 14 to 16

Reviewer 2: ```suggestion
* key stage 4 (KS4) includes years 10 and 11, with pupils age 14 to 16
```

> * key stage 5 (KS5) includes years 12 and 13 from age 16 to 18 at secondary schools with sixth forms or colleges

Reviewer 2: ```suggestion
* key stage 5 (KS5) includes years 12 and 13, from age 16 to 18 at secondary schools with sixth forms or colleges
```

> Pupils work towards national qualifications, usually GCSEs during KS4. Students take A-levels or other similar qualifications during KS5. 

Reviewer: ```suggestion
Pupils work towards national qualifications during KS4, which are usually GCSEs. Students take A levels or other similar qualifications during KS5. 
```

> Pupils work towards national qualifications, usually GCSEs during KS4. Students take A-levels or other similar qualifications during KS5. 

Reviewer 2: ```suggestion
Pupils work towards national qualifications, usually GCSEs during KS4. Students take A levels or other similar qualifications during KS5. 
```
no hyphen according to gov.uk style guide

> Subjects at this level include English, maths and science, computing, physical education (PE) and modern foreign languages. Schools will offer a range of other subjects which could include drama, dance, and media studies. Schools are also required to offer RE, careers guidance and sex and relationship education (SRE).

Reviewer: Need to define what RE is

> Subjects at this level include English, maths and science, computing, physical education (PE) and modern foreign languages. Schools will offer a range of other subjects which could include drama, dance, and media studies. Schools are also required to offer RE, careers guidance and sex and relationship education (SRE).

Reviewer: ```suggestion
Subjects at this level include English, maths and science, computing, physical education (PE) and modern foreign languages. Schools will offer a range of other subjects which could include drama, dance, and media studies. Schools are also required to offer RE, careers guidance, and sex and relationship education (SRE).
```

> Subjects at this level include English, maths and science, computing, physical education (PE) and modern foreign languages. Schools will offer a range of other subjects which could include drama, dance, and media studies. Schools are also required to offer RE, careers guidance and sex and relationship education (SRE).

Reviewer 2: I'm wondering if this list either needs to be exhaustive (which I'm not sure it is), or scaled back a bit? And do we need to point out that the subjects will depend on the school?

e.g.

Subjects at this level include core (?) subjects such as English, maths and science, as well as additional subjects such as drama, dance, and media studies. The subjects offered will depend on the school.

> If you’d prefer to specialise in a subject but still enjoy teaching a variety of things, being a secondary school teacher might suit you.

Reviewer: 'things' feels a bit woolly - can we reword?

### What you need to teach in primary and secondary schools

> To teach in the majority of primary and secondary schools, you will need to achieve qualified teacher status (QTS) and there are a number of routes to this. Check [what training you need to get QTS](/train-to-be-a-teacher).

Reviewer: ```suggestion
To teach in the majority of primary and secondary schools, you will need to achieve qualified teacher status (QTS). There are a number of routes to this. Check [what training you need to get QTS](/train-to-be-a-teacher).
```
Is this last sentence worded quite right compared with where it lands you?

> Once you have QTS, you're qualified to teach any of these age ranges. If you want to change age groups you'll need to be able to show you have relevant classroom experience, for example being able to cover the broader curriculum if you want to switch from secondary to primary.

Reviewer: '**these** age ranges' feels a bit non-specific - which age ranges is this referring to? are we talking about any age ranges in primary and secondary?

> Once you have QTS, you're qualified to teach any of these age ranges. If you want to change age groups you'll need to be able to show you have relevant classroom experience, for example being able to cover the broader curriculum if you want to switch from secondary to primary.

Reviewer: ```suggestion
Once you have QTS, you're qualified to teach any of these age ranges. If you want to change age groups you'll need to be able to show you have relevant classroom experience. For example, if you want to switch from secondary to primary, you will need to show that you can cover the broader curriculum.
```

### How to decide which age group to teach

The best way to work out which age group you would like to teach is to get real life experience in a classroom. 

$get-school-experience$

> If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics then you could do a [paid teaching internship](/teaching-internship-providers) to find out if teaching is right for you.

Reviewer 2: ```suggestion
If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics, then you could do a [paid teaching internship](/teaching-internship-providers) to find out if teaching is right for you.
```

### Pupils with special educational needs

> Most disabled pupils and pupils with special educational needs learn in mainstream schools. Wherever you train and work – whether it’s mainstream schools, special schools, or both – you’ll work with pupils with a range of needs.

Reviewer: ```suggestion
Most disabled pupils and pupils with special educational needs learn in mainstream schools. Wherever you train and work – whether it’s in mainstream schools, special schools, or both – you’ll work with pupils with a range of needs.
```

> You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train and teach.

Reviewer: ```suggestion
You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train to teach.
```
(not quite sure whether this is the right change!)

Reviewer: SEND hasn't been defined at this point so we shouldn't use an acronym?

could change the h3 to add in disabilities (should do this anyway) and then the SEND acronym?

Learn more about the [routes to teach pupils with special educational needs and disabilities (SEND)](/teach-disabled-pupils-and-pupils-with-special-educational-needs).

## Teaching other stages

### Nursery or pre-school
If you want to help young children under 5 years old to develop, you might consider becoming an early years teacher.

Early years teachers play an essential role in children’s development up to the age of 5 and provide high-quality early education.

Learn more about [the routes to become an early years teacher](/early-years-teaching-training).

### Further education
If you’ve got industry experience and a passion for teaching young people over 16 and adults, then further education could be for you. 

> There are three key areas, vocational courses covering areas like plumbing and health care, academic courses on subjects like English and maths, and basic skills courses covering literacy and numeracy. 

Reviewer 2: Does this maybe need a colon and bullets? e.g.

There are 3 key areas:

- vocational etc.
- academic etc.
- basic skills etc.

> There are three key areas, vocational courses covering areas like plumbing and health care, academic courses on subjects like English and maths, and basic skills courses covering literacy and numeracy. 

Reviewer: ```suggestion
There are three key areas: vocational courses covering areas like plumbing and health care, academic courses on subjects like English and maths, and basic skills courses covering literacy and numeracy. 
```
